inc/class-ai-logger.php

```diff
@@ -174,7 +174,14 @@ protected function get_processors(): array {
 		 *
 		 * @param \Monolog\Processor\ProcessorInterface[] $processors Monolog processors.
 		 */
-		return (array) apply_filters( 'ai_logger_processors', [ new \Monolog\Processor\WebProcessor() ] );
+		return (array) apply_filters(
+			'ai_logger_processors',
+			[
+				new \Monolog\Processor\WebProcessor(),
```

Comment:
There seems to be a convention of using `use` statements to avoid the need for fully qualified class references - can that be applied here as well?